charts/camunda-platform/Chart.yaml

```diff
@@ -1,41 +1,59 @@
 apiVersion: v2
 name: camunda-platform
-version: 8.0.11
+version: 8.0.12
 appVersion: "8.0.0"
-description: Camunda Platform Helm Chart for Kubernetes
+description: |
+  Camunda Platform 8 Self-Managed Helm charts.
+  Camunda's process automation platform allows developers to design, automate and improve processes.
 type: application
-icon: https://helm.camunda.io/imgs/self-managed.png
+icon: https://helm.camunda.io/imgs/camunda.svg
 home: https://docs.camunda.io/docs/self-managed/overview/
 sources:
   - https://github.com/camunda/camunda-platform-helm
-  - https://github.com/camunda/zeebe
+keywords:
+  - bpm
+  - bpmn
+  - camunda-platform
+  - dmn
+  - process-engine
+  - workflow
 dependencies:
-- name: zeebe
-  version: 8.0.11
-  condition: "zeebe.enabled"
-- name: zeebe-gateway
-  version: 8.0.11
-  condition: "zeebe.enabled"
-- name: operate
-  version: 8.0.11
-  condition: "operate.enabled"
-- name: tasklist
-  version: 8.0.11
-  condition: "tasklist.enabled"
-- name: optimize
-  version: 8.0.11
-  condition: "optimize.enabled"
-- name: identity
-  version: 8.0.11
-  condition: "identity.enabled"
-- name: elasticsearch
-  repository: "https://helm.elastic.co"
-  version: 7.17.1
-  condition: "elasticsearch.enabled"
+  - name: zeebe
+    version: 8.0.12
+    condition: "zeebe.enabled"
+  - name: zeebe-gateway
+    version: 8.0.12
+    condition: "zeebe.enabled"
+  - name: operate
+    version: 8.0.12
+    condition: "operate.enabled"
+  - name: tasklist
+    version: 8.0.12
+    condition: "tasklist.enabled"
+  - name: optimize
+    version: 8.0.12
+    condition: "optimize.enabled"
+  - name: identity
+    version: 8.0.12
+    condition: "identity.enabled"
+  - name: elasticsearch
+    repository: "https://helm.elastic.co"
+    version: 7.17.1
+    condition: "elasticsearch.enabled"
 maintainers:
   - name: Zelldon
     email: christopher.zell@camunda.com
+  - name: aabouzaid
+    email: ahmed.abouzaid@camunda.com
 annotations:
+  artifacthub.io/links: |
+    - name: Camunda Platform 8 docs
+      url: https://docs.camunda.io/
+    - name: Camunda Platform 8 Self-Managed docs
+      url: https://docs.camunda.io/docs/self-managed/about-self-managed/
+    - name: Camunda Platform 8 Self-Managed Helm charts
+      url: https://github.com/camunda/camunda-platform-helm
   artifacthub.io/changes: |
-    - Add affinity configuration to identity
+    - kind: changed
```

Comment:
:x: the colon is a issue for artifacthub

Reply:
Artifacthub has [2 formats](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for the changes:  structured and non-structured.

this is the structured changelog, and by using it, we will have better highlights in Artifacthub (check the attached screenshots to see how it looks).

![artifacthub-non-structured-changelog](https://user-images.githubusercontent.com/6760103/182804367-89b0c2bd-5baf-491d-9319-9f7910d81318.png)
![artifacthub-structured-changelog](https://user-images.githubusercontent.com/6760103/182804364-f7e4f996-2c11-4b50-86f7-bcdae6f0a2db.png)

Reply:
:+1: Weird last time I got error mails from artifacthub.

Reply:
I think the error shows in the non-structured format where no colon is expected there. But for the structured format, it's part of the structure.